### PR TITLE
Fixed bug in ReadMultipleVars on VarType.Bit

### PR DESCRIPTION
**Pull request by rapha-dev**
Also added BitAdr to Types.DataItem to use Bits in ReadMultipleVars.
Expanded existing UnitTest.
Refer to #115.

### CODE REVIEW

File: S7.Net.UnitTest/S7NetTests.cs [hunk 1]
```diff
@@ -407,19 +430,40 @@ public void T10_ReadMultipleBytes()
...
                     VarType = VarType.Word
-                }
+                },
+                new DataItem
+                {
+                    Count = 1,
+                    DataType = DataType.DataBlock,
+                    DB = 2,
+                    StartByteAdr = 16384,
+                    BitAdr = 6,
```
Comment (mesta1) [inline S7.Net.UnitTest/S7NetTests.cs]: This is undefined.

**Comment by mesta1** [comment]
Can you do the merge?

**Comment by rapha-dev** [comment]
#108 changed PLC response data a little, had to re-test.
Merged to master correctly.

File: S7.Net.UnitTest/S7NetTests.cs (hunk 1 above)
Comment (rapha-dev) [reply S7.Net.UnitTest/S7NetTests.cs]: Added BitAdr to S7.Net/Types/DataItem.cs, was lost in previous merge.

**Comment by mycroes** [comment]
Tested this, seems to work fine. Without this change all results following a single byte read are offset by one byte.

**Comment by mesta1** [comment]
Cool thank you

**Pull request merged**